packages/core/src/http.listener.ts

```diff
@@ -35,5 +35,5 @@ export const httpListener = ({
 
   effect$.subscribe();
 
-  return (req: HttpRequest, res: HttpResponse) => request$.next({ req, res });
+  return (req, res) => request$.next({ req, res });
```

Comment:
Why did you remove types?

Reply:
Because of strict function types for TS compiler. `http.IncomingMessage` does not conform to `HttpRequest` which is a wrapper for it.